actions/sequester/Quest2GitHub/QuestGitHubService.cs

```diff
@@ -227,27 +247,41 @@ private async Task<QuestIteration[]> RetrieveIterationLabelsAsync()
                 });
             }
         }
-        return iterations.ToArray();
+        return [.. iterations];
     }
 
 
-    private async Task<QuestWorkItem?> LinkIssueAsync(string organization, string repo, QuestIssue ghIssue, QuestIteration currentIteration, 
-        IEnumerable<QuestIteration> allIterations)
+    private async Task<QuestWorkItem?> LinkIssueAsync(QuestIssueOrPullRequest ghIssue, QuestIteration currentIteration, IEnumerable<QuestIteration> allIterations)
     {
-        var workItem = LinkedQuestId(ghIssue);
+        int? workItem = LinkedQuestId(ghIssue);
         if (workItem is null)
         {
             // Create work item:
-            var questItem = await QuestWorkItem.CreateWorkItemAsync(ghIssue, _azdoClient, _ospoClient, _areaPath, _importTriggerLabel?.Id, currentIteration, allIterations);
+            QuestWorkItem questItem = await QuestWorkItem.CreateWorkItemAsync(ghIssue, _azdoClient, _ospoClient, _areaPath, _importTriggerLabel?.Id, currentIteration, allIterations);
 
+            string linkText = $"{LinkedWorkItemComment} - {questItem.Id}]({_questLinkString}{questItem.Id})";
+            // For PRs, the body includes a preview table from our build.
+            // The link to the work item should be above it,
+            // with the existing preview table below it.
+            int? previewLinkTableIndex = ghIssue.Body?.IndexOf(PreviewBuildComment);
             // Add Tagged comment to GH Issue description.
-            var updatedBody = $"""
-            {ghIssue.Body}
+            string updatedBody = (previewLinkTableIndex > 0)
```

Comment:
These changes make sure that the "Associated work item" link appears before the internal previews in a PR. The marker comment won't be found in some PRs and all issues.